recipes/tomcat.rb

```diff
@@ -22,6 +22,14 @@
       config(setenv_options)
     end
   end
+
+  apache_tomcat_config 'server' do
+    options do
```

Comment:
In your code:
```
  apache_tomcat_config 'server' do
    options do
      proxy_port node['tomcat']['proxy_port']
      proxy_name node['tomcat']['proxy_name']
      http_scheme node['tomcat']['scheme']
    end
  end
```
I do not see the properties `source` and `cookbook`, just asking if not setting them it's still using your new server.xml template or the original one in the tomcat cookbook.